### DISCUSSION TITLE
Firmware 2.1.17 - MSS310 Smart Plug

### DISCUSSION THREAD

**Discussion by purcell-lab** [krahabb/meross_lan#261] [General]
Meross are inviting me to upgrade to firmware 2.1.17 on my smartplugs, just wondering if there are any experiences out there.

![Screenshot_20230220-121001 (1)](https://user-images.githubusercontent.com/79175134/219993563-e2d31542-ba05-4f76-8b9d-0f1b585f0626.png)

Firmware 2.1.16 is running pretty solid, so I don't feel the need to upgrade urgently.

![image](https://user-images.githubusercontent.com/79175134/219993883-cd1d1c6f-8c76-4914-b36b-313a57bfceb7.png)

**Comment by krahabb** [comment]
Hamletic question...that is!
In my opinion you could skip the update: there have been many reports in the past where an update bringed in an issue (even if then it got fixed rightaway when it was an obvious bug)
Good practive is always to let the 'others' jump in before you so they'll test it in the wild and then after a couple weeks or months of no rumors update your own devices ;)

**Comment by purcell-lab** [comment]
All good.

![image](https://github.com/krahabb/meross_lan/assets/79175134/a442dcb4-241f-4c19-a62b-721dc5aa02db)
